tests/testthat/test-rbackground.R

```diff
@@ -85,3 +85,24 @@ test_that("rpool errors", {
   expect_true(endsWith(ctx$PlainErrorMessage,"rpool_active cannot be set directly"))
 
 })
+
+test_that("reduce cap and active while jobs in queue",{
+  pl$set_options(rpool_cap = 3)
+  l_expr = lapply(1:5,\(i) {
+    pl$lit(i)$map(\(x) {Sys.sleep(.4); -i}, in_background = TRUE)$alias(paste0("lit_",i))
+  })
+  lf = pl$LazyFrame()$select(l_expr)
+  handle = lf$collect(collect_in_background = TRUE)
+  Sys.sleep(.2)
+  pl$set_options(rpool_cap = 2)
+  Sys.sleep(.1)
```

Comment:
> should I do something like "skip if not en var CI='true' "?

Do you mean this?
I think there is probably no problem with setting it like that.

Reply:
Yeah that and potentially other process tests. They have some overhead as they spin-up new processes, and do not touch this too often. Any PR check should still run the tests.